### ISSUE TITLE
Can‘t open file saved from https://www.gcores.com/articles/103310

### ISSUE THREAD

**Issue by OvO7000** [gildas-lormeau/SingleFileZ#45]
when l try to open the file saved from https://www.gcores.com/articles/103310, an error occured.

```
Error: Cannot open the page from the filesystem.
Chrome: Install SingleFileZ and enable the option "Allow access to file URLs" in the details page of the extension (chrome://extensions/?id=offkdfbbigofcgdokjemgjpdockaafjg). Otherwise, start the browser with the switch "--allow-file-access-from-files".
Safari: Select "Disable Local File Restrictions" in the "Develop" menu.
```

l have tried to save file from other pages, there's no problem.   And l have enabled the option "Allow access to file URLs".

**Comment by gildas-lormeau** [comment]
This is a consequence of this regression introduced by Chrome 80 https://bugs.chromium.org/p/chromium/issues/detail?id=1049604. After having reported that regression, Google finally decided it was not a bug but a feature. I'll try to implement a workaround but I'm also considering the removal of SingleFileZ from the store of Google to fix definitely that kind of issues. I recommend you to switch to Firefox.

**Comment by gildas-lormeau** [comment]
I fixed this issue in [this commit](https://github.com/gildas-lormeau/SingleFileZ/commit/87d86003d147f95e922a79fad39a7d3887b814c1). The fix will be available in the version 0.5.37 which should be available for download on the store of Chrome in approximately one week, after having been reviewed by Google.

**Comment by gildas-lormeau** [comment]
The fix has unfortunately an issue. I did a rollback.

**Comment by gildas-lormeau** [comment]
This issue is fixed in the version 0.5.38.
